src/cameras/ArrayCamera.js

```diff
@@ -9,6 +9,7 @@ class ArrayCamera extends PerspectiveCamera {
 		this.isArrayCamera = true;
 
 		this.cameras = array;
+		this.index = 0;
```

Comment:
Is this addition required? It seems there is no read/write access to this property.